src/challenges.js

```diff
@@ -19,11 +25,17 @@ const counter = () => {
   // Example: const newCounter = counter();
   // newCounter(); // 1
   // newCounter(); // 2
+  let count = 0;
+  function increment() {
+    return count++;
+  }
+  return increment;
 };
 
 const limitFunctionCallCount = (cb, n) => {
   // Should return a function that invokes `cb`.
   // The returned function should only allow `cb` to be invoked `n` times.
+
```

Comment:
This method can be done by looking at your previous projects! Take a look at JS-II I believe.
For this portion, you can implement something like a counter and calling a cb from it. If you are still having trouble, just let me know or your group!